.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+ @aperdomob @germandavid85 @luigisamurai @yesidbalvin @Scot3004 @santirogu
```

Comment:
please add to Holmes, Reyes, Valery, and I as codeowners....Those ones are not needed